server/mdm/apple/apple_mdm.go

```diff
@@ -516,28 +516,41 @@ func (d *DEPService) processDeviceResponse(ctx context.Context, depClient *godep
 	for profUUID, serials := range profileToSerials {
 		logger := kitlog.With(d.logger, "profile_uuid", profUUID)
 		level.Info(logger).Log("msg", "calling DEP client to assign profile", "profile_uuid", profUUID)
-		apiResp, err := depClient.AssignProfile(ctx, DEPName, profUUID, serials...)
+
+		skipSerials, assignSerials, err := d.ds.ScreenDEPAssignProfileSerialsForCooldown(ctx, serials)
+		if err != nil {
+			return ctxerr.Wrap(ctx, err, "process device response")
+		}
+		if len(skipSerials) > 0 {
+			// NOTE: the `dep_cooldown` job of the `integrations`` cron picks up the assignments
+			// after the cooldown period is over
+			level.Info(logger).Log("msg", "process device response: skipping assign profile for devices on cooldown", "serials", fmt.Sprintf("%s", skipSerials))
```

Comment:
Just wondering if those are too noisy for `Info` level?  It may very well be fine, just to make sure this was deliberate.

Reply:
It was deliberate as my thinking was that the occurrence of DEP assign profile failures in the wild has been pretty infrequent and something where in the past we've lacked good logging to help in debugging those cases. On the other hand, we're persisting more into the datastore now so moving these to the debug level may make more sense. @roperzh, as someone who has also dealt with debugging these flows in customer environments, do you think it is worth the extra noise here?

Reply:
though choice! agreed with everything you said Sarah, having something like this would have been huge, but now we have an UI indication, and a DB record, etc.

moving it to debug might be a good compromise? it's enabled in cloud IIRC